README.md - "Developer setup info" is not clear

I think these will encourage admin level user to try out what's coming next in Mail app.

1. please provide non-distribution specified instructions (with whatever distribution specified instructions you guys using).

2. provide external lib download links (for distributions without said lib package)

3. instructions should be working

@ChristophWurst I see this error running as root:

```
> mail@0.3.0 bower_deps /mail
> ./node_modules/.bin/bower install

bower ESUDO         Cannot be run with sudo

Additional error details:
Since bower is a user command, there is no need to execute it with superuser permissions.
If you're having permission errors when using bower without sudo, please spend a few minutes learning more about how your sys                                tem should work and make any necessary repairs.

http://www.joyent.com/blog/installing-node-and-npm
https://gist.github.com/isaacs/579814

You can however run a command with sudo using --allow-root option
npm ERR! weird error 1
npm WARN This failure might be due to the use of legacy binary "node"
npm WARN For further explanations, please read
/usr/share/doc/nodejs/README.Debian

npm ERR! not ok code 0
Script npm run-script bower_deps handling the post-install-cmd event returned with an error



  [RuntimeException]
  Error Output: bower ESUDO         Cannot be run with sudo
  Additional error details:
  Since bower is a user command, there is no need to execute it with superuser permissions.
  If you're having permission errors when using bower without sudo, please spend a few minutes learning more about how you
  r system should work and make any necessary repairs.
  http://www.joyent.com/blog/installing-node-and-npm
  https://gist.github.com/isaacs/579814
  You can however run a command with sudo using --allow-root option
  npm ERR! weird error 1
  npm WARN This failure might be due to the use of legacy binary "node"
  npm WARN For further explanations, please read
  /usr/share/doc/nodejs/README.Debian

  npm ERR! not ok code 0
```

What should I do?

--- a/README.md
+++ b/README.md
@@ -83,6 +83,16 @@ php composer.phar install
 ```
 Debian/Ubuntu people, just execute `install_ubuntu.sh`
 
+### Nightly builds
+
+Instead of setting everything up manually, you can just [download the nightly builds](https://en0ch.se/index.php/s/3v5J3XXeQCCp73p) instead. These builds are updated every 12 hours - 00:00 UTC+0200 (12:00 AM) and 12:00 UTC+0200 (12:00 PM), and are pre-configured with all the needed dependencies.
+
+1. Download
+2. Extract the zip archive to 'path-to-owncloud/apps'
+3. Navigate to »Apps«, choose the category »Productivity«, find the Mail app and enable it.
+
+The nightly builds are provided by [Daniel Hansson](https://github.com/enoch85)
+
 ### Resetting the app
 Connect to your database and run the following commands (`oc_` is the default table prefix):
 ```sql

